praktikumsplaner-backend/src/main/resources/application-dev.yml

```diff
@@ -13,13 +13,7 @@ spring:
   h2.console.enabled: true
   jpa:
     database: H2
-    hibernate:
-        # always drop and create the db should be the best
-        # configuration for local (development) mode. this
-        # is also the default, that spring offers by convention.
-        # but here explicite:
-        ddl-auto: create-drop
-        naming.physical-strategy: org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
```

Comment:
are u sure we can delete the Naming-Strategy? I had some trouble with `dueDate` in bei TODO-App. Without the strategy, hibernate was looking for a column `due_date` but my column was `dueDate`.

Reply:
Moved to application.yml